main.css

```diff
@@ -0,0 +1,182 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
Remove margin and padding here. I think you just want to remove the default margin on the body, so move margin: 0 to the body selector